### PR TITLE
minor borg and ai fixes

### PR DESCRIPTION
**Pull request by TheReduce**
## Changelog
:cl:

fix: now ai properly states laws in channels
fix: now cyborg can choose type of floor

/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 1]
```diff
@@ -117,6 +117,44 @@
...
+	"white floor tile" = /obj/item/stack/tile/floor/white,
+	"white cargo tile" = /obj/item/stack/tile/floor/white/cargo,
+	"red carpet" = /obj/item/stack/tile/carpet,
+	"black carpet" = /obj/item/stack/tile/carpet/bcarpet,
+	"blue carpet" = /obj/item/stack/tile/carpet/blucarpet,
+	"turquoise carpet" = /obj/item/stack/tile/carpet/turcarpet,
+	"silver blue carpet" = /obj/item/stack/tile/carpet/sblucarpet,
+	"purple carpet" = /obj/item/stack/tile/carpet/purcarpet,
+	"orange carpet" = /obj/item/stack/tile/carpet/oracarpet
+)
```
Comment (LethalGhost) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: Isn't it better to makt it static internal variable for `/obj/item/stack/tile/floor/cyborg/`?

Comment (TheReduce) [reply code/game/objects/items/stacks/tiles/tile_types.dm]: https://github.com/discordia-space/CEV-Eris/pull/4226/commits/c1aa7470d950be03f67fda7f7e17f02eb2a4d162

File: code/modules/mob/living/silicon/laws.dm [hunk 2]
```diff
@@ -79,7 +79,7 @@
 
 	stating_laws[prefix] = 1
 
-	var/can_state = statelaw("[prefix]Current Active Laws:")
+	var/can_state = statelaw("[prefix] Current Active Laws:")
```
Comment (ACCount12) [inline code/modules/mob/living/silicon/laws.dm]: Why is this necessary?
